kafka-plugins-0.8/pom.xml

```diff
@@ -5,13 +5,13 @@
   <parent>
     <artifactId>kafka-plugins</artifactId>
     <groupId>co.cask.hydrator</groupId>
-    <version>2.0.1</version>
+    <version>2.0.2-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
   <name>Apache Kafka 0.8 plugins</name>
   <artifactId>kafka-plugins</artifactId>
-  <version>${project.parent.version}-0.8.2.2</version>
+  <version>2.0.2-SNAPSHOT</version>
```

Comment:
need to preserve the kafka version in the version

Reply:
Thanks. Pushed update.